tests/tests_unit/test_db/test_db_reader_unit/test_kafka_reader_unit.py

```diff
@@ -144,3 +144,21 @@ def test_kafka_reader_invalid_hwm_column(spark_mock, hwm_column):
             table="table",
             hwm_column=hwm_column,
         )
+
+
+def test_kafka_connection_with_parameters_for_batch_strategy(spark_mock):
```

Comment:
```suggestion
def test_kafka_connection_with_parameters_for_batch_strategy_error(spark_mock):
```

падающий тест проще видеть сразу если error в имени